<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="EC2AMAZ-EC8CRPV" failures="1" tests="1" name="Default test(failed)" time="0.063" errors="0" timestamp="2021-11-22T05:10:13 GMT">
  <testcase classname="demo.TestNGRerunDemo" name="test2" time="0.009">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at demo.TestNGRerunDemo.test2(TestNGRerunDemo.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="demo.TestNGRerunDemo" name="test3" time="0.0">
    <ignored/>
  </testcase> <!-- test3 -->
  <testcase classname="demo.TestNGRerunDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
